tests/dm_syncer/conf/diff_config.toml

```diff
@@ -16,17 +16,57 @@ fix-sql-file = "fix.sql"
 
 # tables need to check.
 [[check-tables]]
-schema = "dm_syncer"
-tables = ["~t.*"]
+schema = "forum"
```

Comment:
we might keep original "dm_syncer". to test another database, we could copy this file such as `diff_config_balist.toml`, and use it in `run.sh` of dm_syncer like

```
check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
check_sync_diff $WORK_DIR $cur/conf/diff_config_balist.toml
```